service/controller/v17/encrypter/vault/vault.go

```diff
@@ -74,134 +73,6 @@ func NewEncrypter(c *EncrypterConfig) (*Encrypter, error) {
 	return e, nil
 }
 
```

Comment:
Unfortunately this had to be done in two implementations at once, because of the interface.